iaso/models/base.py

```diff
@@ -1039,6 +1039,8 @@ def as_full_model(self):
             "longitude": self.location.x if self.location else None,
             "altitude": self.location.z if self.location else None,
             "period": self.period,
+            "planning_id": self.planning.id if self.planning else None,
+            "planning_name": self.planning.name if self.planning else None,
```

Comment:
you can probably add team id here and avoid previous calls

Reply:
Yep, that looks like the better option, since you're already adding fields